tests/capstone2llvmir/x86_tests.cpp

```diff
@@ -9501,6 +9501,466 @@ TEST_P(Capstone2LlvmIrTranslatorX86Tests, X86_INS_CMOVS_no_move)
 	EXPECT_NO_VALUE_CALLED();
 }
 
+//
+// X86_INS_FCMOVB
+//
+
+TEST_P(Capstone2LlvmIrTranslatorX86Tests, X86_INS_FCMOVB_move_1)
```

Comment:
Add comments for each FPU instruction test you added.
Comment = instruction description from the sites like:
https://www.felixcloutier.com/x86/fcmovcc.
https://c9x.me/x86/html/file_module_x86_id_81.html
Look at other FPU instructions.
Make sure, all the opcode variants from tables have at least one test.

E.g.
```cpp
// DA /0	FIADD m32int	Add m32int to ST(0) and store result in ST(0).
TEST_P(Capstone2LlvmIrTranslatorX86Tests, X86_INS_FIADD_m32) { /*...*/}

// DE /0	FIADD m16int	Add m16int to ST(0) and store result in ST(0).
TEST_P(Capstone2LlvmIrTranslatorX86Tests, X86_INS_FIADD_m16) { /*...*/}
```